doc/third-party/fivetran/setup-guide.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Setup Guide
+title: Materialize Setup Guide
+description: Follow the guide to set up Materialize as a destination.
+---
+
+# Materialize Setup Guide {% badge text="Partner-Built" /%} {% badge text="Private Preview" /%}
+
+Follow the setup guide to connect Materialize to Fivetran.
+
+> NOTE: This destination is [partner-built](/docs/partner-built-program). For any questions related to Materialize destination and its documentation, contact [Materialize Support](mailto:support@materialize.com).
+
+---
+
+## Prerequisites
+
+To connect Materialize to Fivetran, you need the following:
+
+- A Materialize account with `CREATE` privileges on the database you're connecting to.
+- A Fivetran account with [permission to add destinations](/docs/using-fivetran/fivetran-dashboard/account-management/role-based-access-control#legacyandnewrbacmodel).
+
+---
+
+## Setup instructions
+
+### <span class="step-item"> Get Materialize credentials </span> 
+
+You must have a Materialize account with `CREATE` privileges on the database you're connecting to.
+
+1. Login to the [Materialize Console](https://console.materialize.com) with your account.
+2. Click "Connect externally" at the bottom of the sidebar.
+3. On the "External tools" tab, click "Create app password". 
+4. Copy the generated **password**, **Host**, and **User** somewhere safe, we'll use these soon!
+
+### <span class="step-item">Finish Fivetran configuration </span>
+
+1. Log in to your Fivetran account.
+2. Go to the [**Destinations** page](https://fivetran.com/dashboard/destinations), and then click **+ Add Destination**.
```

Comment:
I know you copied this from the example, but the **Add Destination** button doesn't actually have a "+" anywhere, not sure why the example wrote it that way.